services/user/Sites/src/Sites.cpp

```diff
@@ -53,6 +83,125 @@ namespace SystemService
             target.resize(pos);
          return target;
       }
+
+      /*
+        script-src:
+          * `unsafe-eval` is needed for WebAssembly.compile()
+          * `unsafe-inline` is needed for inline <script> tags
+          * `blob:` for loading dynamically generated modules
+          * `https:` for dynamically loading libs from CDNs
+        font-src:
+          * `https:` for dynamically loading fonts from CDNs
+        frame-src:
+          * `*` is needed to allow embedding supervisor
+        connect-src:
+          * `blob:` for fetch-compiling blob urls
+          * `*` allows fetching plugins, as well as websocket connections
+      */
+      const std::string DEFAULT_CSP_HEADER =                               //
+          "default-src 'self';"                                            //
+          "font-src 'self' https:;"                                        //
+          "script-src 'self' 'unsafe-eval' 'unsafe-inline' blob: https:;"  //
+          "img-src *;"                                                     //
+          "style-src 'self' 'unsafe-inline';"                              //
+          "frame-src *;"                                                   //
+          "connect-src * blob:;"                                           //
+          ;
+
+      // Accepted content encodings
+      const std::array<std::string, 2> validEncodings = {"br", "gzip"};
```

Comment:
I prefer built-in arrays for this, just because the size can be deduced. Also, using `string_view` instead of string allows static initialization.